.env.default

```diff
@@ -5,3 +5,5 @@ FLASK_APP=multinet
 FLASK_ENV=development
 
 MYPYPATH=mypy_stubs
+
+SENTRY_DSN=
```

Comment:
These lines don't seem necessary due to how `os.getenv()` is called during app startup. Is this just to show the general structure of this file?

Reply:
Yeah that was my main reason. If we don't like this we could just list this in the docs, along with other env vars if we're not currently listing them.

Reply:
No worries, we can leave this as is.